sdcm/sct_events/database.py

```diff
@@ -85,6 +87,10 @@ def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_even
                                    regex="filesystem_error")
 DatabaseLogEvent.add_subevent_type("STACKTRACE", severity=Severity.ERROR,
                                    regex="stacktrace")
+
+# REACTOR_STALLED must be above BACKTRACE as it has "Backtrace" in its message
+DatabaseLogEvent.add_subevent_type("REACTOR_STALLED", mixin=ReactorStalledMixin, severity=Severity.DEBUG,
+                                   regex="Reactor stalled")
```

Comment:
The initial idea was to not use DEBUG level in a code, isn't it? 

```python
class Severity(enum.Enum):
    DEBUG = -1  # this should only be through test case configuration!!!
...
```

Reply:
yes, but that was the name convention agreed by @roydahan @bentsi and @fruch

Reply:
Yes, it was agreed to have DEBUG.